src/modules/75-cf/components/ListingPageTemplate/ListingPageTemplate.tsx

```diff
@@ -62,15 +66,30 @@ const ListingPageTemplate: React.FC<ListingPageTemplateProps> = ({
     [title, titleTooltipId]
   )
 
+  const { getString } = useStrings()
+
   return (
     <main className={css.layout}>
       <Page.Header
         title={headerTitle}
         breadcrumbs={<NGBreadcrumbs customPathParams={{ module: 'cf' }} links={breadcrumbs} />}
-        className={css.header}
+        className={cx(css.header, headerLinks && css.withLinks)}
         content={headerContent}
       />
 
+      {headerLinks?.docsURL && (
+        <Page.SubHeader className={css.links}>
+          <a href={headerLinks.docsURL} target="_blank" rel="noopener noreferrer">
+            <Layout.Horizontal flex={{ alignItems: 'center', justifyContent: 'flex-start' }} spacing="xsmall">
+              <Icon name="learning" color={Color.PRIMARY_7} />
+              <Text font={{ variation: FontVariation.BODY }} color={Color.PRIMARY_7} tag="div">
+                {getString('cf.shared.readDocumentation')}
+              </Text>
+            </Layout.Horizontal>
```

Comment:
I think you might be able to just use the `icon` and `iconProps` props in the `Text` component to combine all this stuff.
```suggestion
            <Text
              font={{ variation: FontVariation.BODY }}
              color={Color.PRIMARY_7}
              tag="div"
              icon="learning"
              iconProps={{ color: Color.PRIMARY_7 }}
            >
              {getString('cf.shared.readDocumentation')}
            </Text>
```

Reply:
Yeah I'd done this originally when using the SVG icon, that has since been removed